babel.config.js

```diff
@@ -4,4 +4,5 @@ module.exports = {
     "@babel/preset-typescript",
     "ts-node"
   ],
+  sourceMap: true,
```

Comment:
was this intentional?

Reply:
No. Useful for debugging, it won't hurt to provide it in another version, alongside with an update away from webpack to unify how we build packages in our JS SDKs